### PR TITLE
Installed PrimeVue, tested Button and InputText componets, icons, styles

### PR DESCRIPTION
**Pull request by markuwas**
# Testing PrimeVue
Installed PrimeVue - https://www.primefaces.org/primevue/
Tested components: Button, InputText
Tested icons and styles

### CODE REVIEW

**Comment by maveric-ui** [comment]
updated branch
remove all testes code, leave only packages and configurations

**Comment by markuwas** [comment]
Test code removed

**Comment by maveric-ui** [comment]
@markuwas resolve conflicts

File: src/main.js [hunk 1]
```diff
@@ -4,8 +4,15 @@ import store from './store';
 
 import App from './App.vue';
 
+import PrimeVue from 'primevue/config';
+
+import 'primevue/resources/themes/saga-blue/theme.css';
```
Comment (maveric-ui) [inline src/main.js]: move it to global styles

**Review by tsehla42 — DISMISSED** [review]
LGTM!

**Pull request closed without merging**